types/react-select/src/styles.d.ts

```diff
@@ -31,11 +31,6 @@ import { CSSProperties } from 'react';
 
 export interface Props { [key: string]: any; }
 
-/**
- * @param base -- the component's default style
- * @param state -- the component's current state e.g. `isFocused`
- * @returns
- */
```

Comment:
Removed these lines as they were causing lint errors due to the no-redundant-jsdoc rule.